charts/fybrik/templates/fybrik-blueprints-role.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: rbac.authorization.k8s.io/v1
 kind: Role
```

Comment:
Not related to this review, but we will need to address and change using verb '*' and resources '*' at some point. This is be a security issue to use *.